applications/server/controllers/synchronization.py

```diff
@@ -1,12 +1,8 @@
 # -*- coding: utf-8 -*-
 import json
 
-
-
-@request.restful()
-def receive_logs():
-    """
-    This method is to recieve log data.  It accepts two HTTP methods:
+"""
+ This method is to recieve log data.  It accepts two HTTP methods:
         GET: returns the latest three log entries.  planning on adding
```

Comment:
Idea: can you make this the method that the Web UI people call to do the visualization?